core/src/main/scala/org/apache/spark/shuffle/sort/SortShuffleManager.scala

```diff
@@ -75,9 +77,10 @@ private[spark] class SortShuffleManager(conf: SparkConf) extends ShuffleManager
   }
 
   /**
-   * A mapping from shuffle ids to the number of mappers producing output for those shuffles.
+   * A mapping from shuffle ids to the tuple of number of mappers producing output and
+   * indeterminate stage attempt id for those shuffles.
```

Comment:
`indeterminate stage attempt id` -> `shuffle generation id`

Reply:
Thanks, done in 96bab1e.